menas-web/ui/components/login/loginDetail.controller.js

```diff
@@ -140,7 +149,7 @@ sap.ui.define([
         this.byId(usernameField).setValueState(sap.ui.core.ValueState.Error);
         this.byId(passwordField).setValueState(sap.ui.core.ValueState.Error);
       };
-      $.ajax("api/login", {
+      $.ajax(window.apiUrl + "/api/login", {
```

Comment:
If it's called `apiUrl` why to add the subfolder `api` everywhere?

Reply:
Good point. As an idea, should Menas also keep the `/api` paths?